doc/node.1

```diff
@@ -213,6 +213,12 @@ Specify the
 .Ar module
 to use as a custom module loader.
 .
+.It Fl -insecure-http-parser
+Use an insecure HTTP parser that accepts invalid HTTP headers. This may allow
+interoperability with non-conformant HTTP implementations. It may also allow
+request smuggling and other HTTP attacks that rely on invalid headers being
+accepted.
```

Comment:
```suggestion
accepted. Avoid using this option.
```

Reply:
If you are going to suggest to avoid using the feature, then it should also explain the reason for its existence in the first place, what are the security implications of using this feature, and how they can be mitigated.

Reply:
Never mind. It is already said in the above paragraph. I still think that "Avoid using this option." does not add anything of value to the conversation.

Reply:
Yeah, I only suggested it for parity.

Reply:
The node.1 text should be same as the cli.md, thanks for noticing. I added the text on explicit request, see review history.